src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -152,6 +209,38 @@ class ZHAConfigDashboard extends LitElement {
     this._configuration = await fetchZHAConfiguration(this.hass!);
   }
 
+  private async _fetchSettings(): Promise<void> {
+    this._networkSettings = await fetchZHANetworkSettings(this.hass!);
+  }
+
+  private async _createAndDownloadBackup(): Promise<void> {
+    this._generatingBackup = true;
+    const backup_and_metadata: ZHANetworkBackupAndMetadata =
+      await createZHANetworkBackup(this.hass!);
+    this._generatingBackup = false;
```

Comment:
Put this in a `finally` so that it is always set to `false`, even if `createZHANetworkBackup` throws an exception.

Reply:
Should I display an error message if an error is thrown or is just a `try...finally` block fine?

Reply:
I think a message that it failed would also be good.